roles/wordpress/vars/main.yml

```diff
@@ -76,7 +82,7 @@ wordpress_backend_server:
       text/x-cross-domain-policy
       text/xml;
 
-    rewrite /wp-admin$ $scheme://$host$uri/ permanent;
+    rewrite ^/wp-admin$ $scheme://$host$uri/ permanent;
```

Comment:
This will break for subdirectory multisite installations. That's why I left the rewrite rule as is.